config/jobs/kubernetes/sig-release/release-branch-jobs/1.14.yaml

```diff
@@ -165,36 +165,41 @@ periodics:
       securityContext:
         privileged: true
 - annotations:
+    testgrid-alert-email: kubernetes-sig-testing-alerts@googlegroups.com, k8s-infra-oncall@google.com
     testgrid-dashboards: sig-release-1.14-blocking, google-unit
     testgrid-tab-name: bazel-build-1.14
+  decorate: true
+  extra_refs:
+  - base_ref: release-1.14
+    org: kubernetes
+    repo: kubernetes
+  - base_ref: master
+    org: kubernetes
+    repo: test-infra
   interval: 6h
   labels:
-    preset-bazel-remote-cache-enabled: "true"
     preset-bazel-scratch-dir: "true"
     preset-service-account: "true"
   name: periodic-kubernetes-bazel-build-1-14
   spec:
     containers:
     - args:
-      - --job=$(JOB_NAME)
-      - --repo=k8s.io/kubernetes=release-1.14
-      - --service-account=/etc/service-account/service-account.json
-      - --upload=gs://kubernetes-jenkins/logs
-      - --timeout=60
-      - --scenario=kubernetes_bazel
-      - --
-      - --build=//... -//vendor/...
-      - --release=//build/release-tars
-      env:
-      - name: REPO_OWNER
-        value: kubernetes
-      - name: REPO_NAME
-        value: kubernetes
+      - -c
+      - |
+        set -o errexit
+        gcloud auth activate-service-account --key-file=/etc/service-account/service-account.json
+        ../test-infra/scenarios/kubernetes_bazel.py \
+            --config=ci \
+            "--build=//... -//vendor/..." \
+            --release=//build/release-tars \
+            --gcs=gs://kubernetes-release-dev/ci \
+            --version-suffix=-bazel \
+            --publish-version=gs://kubernetes-release-dev/ci/latest-bazel-1.14.txt
+      command:
+      - bash
```

Comment:
(see above)

Reply:
Can you describe the functional difference between this added block, and the one above?